mcs/Makefile

```diff
@@ -26,6 +26,28 @@ orbis_SUBDIRS := build class
 
 include build/rules.make
 
+monodroid_PLATFORMS := android
```

Comment:
I will move it to `build/profiles/$(PROFILE).make`. For profiles that only run on 1 platform, it just makes it much easier to still provide the 1 platform, so we can generalise the rest of the build system, and not special case for when the platform is provided or not.
It also opens the door (at a  later time) for what @kumpera suggested of generalising profiles (for example: orbis becomes monotouch + orbis platform or something equivalent).